locale/compile-mo.sh

```diff
@@ -41,7 +41,7 @@ for pofile in `find $1 -type f -name "djangojs.po"`; do
     lang=`echo $pofile | cut -d "/" -f2`
     stem=`basename $pofile .po`
     touch "${dir}/${stem}.mo"
-    if [ $lang != 'dbg' ]
+    if [ $lang != 'dbg' ] && [ $lang != 'dbr' ] && [ $lang != 'dbl' ]
```

Comment:
Later on you've removed `dbg` and replaced with `dbr` and `dbl` but you've kept it here?  Intentional?